index.html

```diff
@@ -381,6 +391,71 @@ <h2 class="contact-heading">Contact me</h2>
         </div>
       </section>
     </div>
-    <script defer src="./index.js" type="module"></script>
+    <script defer src="./script/index.js" type="module"></script>
+
+    <!-- General Card for Each popup Window  -->
+    <div class="wrapper">
+      <div id="card1" class="card-works popup-card">
+        <div class="left-block">
+          <div class="title-box">
+            <div class="project-top">
+              <h2 class="project-title" id="card-title">Tonic</h2>
+              <i class="bi bi-x-lg" id="close-project"></i>
+            </div>
+            <div class="canopy">
+              <div class="client" id="card-client">CANOPY</div>
+              <div class="counter"></div>
+              <div class="role" id="card-role">Back End Dev</div>
+              <div class="counter"></div>
+              <div class="year" id="card-year">2015</div>
+            </div>
+          </div>
+        </div>
+
+        <img
+          src="./images/work/Tonic.png"
+          alt="Tonic Image"
+          class="snapshot-portfolio"
+          id="mobile-image"
+        />
+        <img
+          src="./images/desktop/Portoflio_Card/nature.png"
+          alt="Nature"
+          class="desktop-portfolio-img"
+          id="desktop-image"
+        />
+
+        <div class="left-block bottom-box">
+          <p class="description-box" id="card-description">
+            Lorem Ipsum is simply dummy text of the printing and typesetting
+            industry. Lorem Ipsum has been the industry's standard dummy text
+            ever since the 1500s, when an unknown printer took a galley of type
+            and scrambled it to make a type specimen book. It has survived not
+            only five centuries, but also the leap into electronic typesetting,
+            remaining essent
+          </p>
+          <div class="righ-box">
+            <div class="tag-box">
+              <ul class="tags" id="card-tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">Javascript</li>
+              </ul>
+            </div>
+            <div class="hr-line"></div>
+            <div class="action-box">
+              <button class="btn" type="button">
+                See Live
+                <img src="./images/project/seelive.png" alt="See Live Image" class="btn-images">
+              </button>
+              <button class="btn" type="button">
+                See Source
+                <img src="./images/project/github.png" alt="Github Image" class="btn-images">
+              </button>

```

Comment:
- [ ] please try to remove this project cause you should generate cards dynamically please try to fix that